lib/browser/api/menu-item.js

```diff
@@ -6,15 +6,16 @@ nextCommandId = 0
 
 // Maps role to methods of webContents
 rolesMap = {
-  undo: 'undo',
-  redo: 'redo',
-  cut: 'cut',
-  copy: 'copy',
-  paste: 'paste',
-  selectall: 'selectAll',
-  minimize: 'minimize',
-  close: 'close',
-  delete: 'delete'
+  'undo': 'undo',
+  'redo': 'redo',
+  'cut': 'cut',
+  'copy': 'copy',
+  'paste': 'paste',
+  'paste-and-match-style': 'pasteAndMatchStyle',
```

Comment:
Went with the same name used when implementing for OS X
https://github.com/electron/electron/blob/master/atom/browser/ui/cocoa/atom_menu_controller.mm#L36